src/unittest/python/__init__.py

```diff
@@ -0,0 +1 @@
+# Name: Debasish Kanhar
```

Comment:
Copyright header here and in other __init__.py scripts below

Reply:
Do we need copyright statements in unit tests too?

Reply:
Added copyrights in all unit tests files @sjudeng , but I'm still in doubt should we be adding those? I mean yes those codes are going to be hosted on Github repo, but won't be part of distribution which will be installed using PyPi. i.e. the final built versions/libraries won't contain these files.

Reply:
Thanks. I think test files do usually include copyright/license headers (for example see ([janusgraph-test](https://github.com/JanusGraph/janusgraph/tree/master/janusgraph-test) and [gremlin-python](https://github.com/apache/tinkerpop/tree/master/gremlin-python/src/test/java/org/apache/tinkerpop/gremlin/python))).